cmd/manager/main.go

```diff
@@ -159,7 +157,7 @@ func newRootCommand() *cobra.Command {
 			log.Info("Starting /healthz and /readyz endpoints")
 			go http.ListenAndServe(":8080", nil)
 
-			run := func(ctx context.Context) {
+			run := func(ctx context.Context, cancel context.CancelFunc) {
```

Comment:
The cancel function doesn't need to be passed in as a parameter. Create the context for the leader election prior to creating this function.

Reply:
addressed